src/main/resources/static/css/matchresult.less

```diff
@@ -1,5 +1,7 @@
-@import "_colorscheme.less";
-@import "searchbar.less";
+@import "_colorscheme";
+@import "_mediaqueries";
+@import "_background";
+@import "_searchbar";
```

Comment:
I prefer that the filenames are written out - including the filetype.
We already have that convention for exampple in the controllers where the `.html` ending specifys the type.
So does everyody know what kind of file this is.

Please append them to the other files too, i won't comment everyone of them.

Reply:
Everything else looks fine to me

Reply:
> 
> 
> Why do some files begin with an underscore? And wouldn't this PR close #238?

Files containing meta information meaning that there are not used by a 'physical' site begin with an underscore to show that. Less will compile them an insert the rules via the @import